src/main/scala/amailp/intellij/robot/psi/reference/LibraryToDefinitionReference.scala

```diff
@@ -0,0 +1,55 @@
+package amailp.intellij.robot.psi.reference
+
+import com.intellij.codeInsight.lookup.{AutoCompletionPolicy, LookupElementBuilder}
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi._
+import com.intellij.psi.util.QualifiedName
+import com.jetbrains.python.psi.{PyClass, PyFile}
+import com.jetbrains.python.psi.stubs.{PyClassNameIndex, PyModuleNameIndex}
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+import amailp.intellij.robot.psi.utils.ExtRobotPsiUtils
+
+
+class LibraryToDefinitionReference(element: PsiElement, textRange: TextRange) extends PsiReferenceBase[PsiElement](element, textRange)
+  with PsiPolyVariantReference with ExtRobotPsiUtils {
+
+  override def getVariants = {
```

Comment:
the getVariants serves the purpose of populate the autocompletion list. What is the point in populating it with the list of the already imported libraries?
Theoretically all the reachable python modules and classes are candidates for completion but the ones already imported are the less interesting, don't you think?
I would rather return an empty result...